modules/ROOT/pages/siebel/siebel-connector.adoc

```diff
@@ -177,9 +177,9 @@ The following example shows the `Loggers` element with the `AsyncLogger` line ad
 		</Console>
 	</Appenders>
 	<Loggers>
-		<Logger name="org.mule.modules.siebel.internal.service" level="debug" additivity="false">
+		<AsyncLogger name="org.mule.modules.siebel.internal.service" level="DEBUG" additivity="FALSE">
```

Comment:
`level="DEBUG"` is fine. But I haven't found evidence to confirm that `additivity="FALSE"` will, so I'd change that back to `false`.